Api/Services/ReviewService.cs

```diff
@@ -140,5 +144,95 @@ public async Task<Result<ReviewVM>> GetReviewAsync(int reviewId)
 
             return review;
         }
+    
+        /// <summary>
+        /// Update restaurant response by review ID
+        /// </summary>
+        /// <param name="reviewId">ID of the review</param>
+        /// <param name="user">Current user for permission checking</param>
+        /// <param name="userId">Id of the current user for permission checking</param>
+        /// <param name="restaurnatResponse">New restaurant response</param>
+        /// <returns></returns>
+        [ErrorCode(null, ErrorCodes.NotFound)]
+        [ErrorCode(null, ErrorCodes.AccessDenied)]
```

Comment:
widzę że `VerifyOwnerRole` nie ma `ErrorCode`. przenieś prosze to do `VerifyOwnerRole`, a tu użyj `MethodErrorCodes`